examples/sample-pub/publisher.js

```diff
@@ -144,6 +144,41 @@ function startFlowAuto() {
     return;
   }
 
+  if (flow == 'metering') {
+    whenReady((subscriptions) => {
+      subscriptions
+        .getEntitlements({
+          metering: {
+            state: {
+              id:
+                'user5901e3f7a7fc5767b6acbbbaa927d36f5901e3f7a7fc5767b6acbbbaa927',
+              standardAttributes: [
+                {
+                  name: 'registered_user', // standard_registered_user
+                  timestamp: Math.floor(Date.now() / 1000),
```

Comment:
This time must not change (unless the date the user registered changed :-).  Using `Date.now()` makes it appear that user data has changed on every page.   The timestamp is the time the attribute last changed.

Reply:
I added a variable that defines a specific time.
`const timestamp = 1597686771`